theme/ckeditor5-ui/components/inputtext/inputtext.css

```diff
@@ -23,6 +23,9 @@
 	/* This is important to stay of the same height as surrounding buttons */
 	min-height: var(--ck-ui-component-min-height);
 
+	transition-property: box-shadow,border;
+	transition: .2s ease-in-out;
```

Comment:
Maybe we should variable for duration and/or timing-function like `--ck-input-text-transition: .2s ease-in-out`

I really like that transition! WDYT about using a shorthand CSS syntax?
```css
transition: box-shadow, border var(--ck-input-text-transition);
```

Reply:
That's for sure. But TBH, I'd do that in https://github.com/ckeditor/ckeditor5/issues/810 because most likely we'd have to figure out a set of variables to control transitions/animations across the theme anyway.